--- eog-2.25.92/configure.ac~	2009-03-15 18:12:32.222616928 +0100
+++ eog-2.25.92/configure.ac	2009-03-15 18:13:28.487391837 +0100
@@ -361,11 +361,11 @@
 	AC_MSG_RESULT([$PYGTK_DEFSDIR])
 
 	AC_MSG_CHECKING([for pygobject codegen])
-	PYGOBJECT_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygobject-2.0`/codegen.py"
+	PYGOBJECT_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygobject-2.0`/codegen.pyc"
 	AC_MSG_RESULT([$PYGOBJECT_CODEGEN])
 
 	AC_MSG_CHECKING([for pygobject h2def])
-	PYGOBJECT_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygobject-2.0`/h2def.py"
+	PYGOBJECT_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygobject-2.0`/h2def.pyc"
 	AC_MSG_RESULT([$PYGOBJECT_H2DEF])
 
 	AC_SUBST([PYGOBJECT_DEFSDIR])
