diff -urN eog-1.1.0.orig/viewer/Makefile.am eog-1.1.0/viewer/Makefile.am
--- eog-1.1.0.orig/viewer/Makefile.am	Sun Oct 13 23:44:47 2002
+++ eog-1.1.0/viewer/Makefile.am	Sun Oct 13 23:48:30 2002
@@ -36,7 +36,7 @@
 #	 eog-embeddable.c	\
 #	 eog-embeddable.h	\
 #	 eog-embeddable-view.c	\
-#	 eog-embeddable-view.h	\
+#	 eog-embeddable-view.h	
 
 eog_image_viewer_DEPENDENCIES = $(top_builddir)/libeog/libeog.a
 
