diff -Nuard eog-2.4.1.orig/shell/eog-window.c eog-2.4.1/shell/eog-window.c
--- eog-2.4.1.orig/shell/eog-window.c	2003-07-16 17:35:07.000000000 +0200
+++ eog-2.4.1/shell/eog-window.c	2004-02-24 21:39:04.225051216 +0100
@@ -416,6 +416,7 @@
 	}
 
 	if (priv->ctrl_frame != NULL) {
+		bonobo_control_frame_control_deactivate(BONOBO_CONTROL_FRAME(priv->ctrl_frame));
 		bonobo_object_unref (BONOBO_OBJECT (priv->ctrl_frame));
 		priv->ctrl_frame = NULL;
 	}
