--- eog-2.19.91/configure.ac~	2007-08-27 19:32:07.000000000 +0200
+++ eog-2.19.91/configure.ac	2007-08-29 08:44:18.661755053 +0200
@@ -305,11 +305,11 @@
 	AC_MSG_RESULT([$PYGTK_DEFSDIR])
 
 	AC_MSG_CHECKING([for pygtk codegen])
-	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.py"
+	PYGTK_CODEGEN="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/codegen.pyc"
 	AC_MSG_RESULT([$PYGTK_CODEGEN])
 
 	AC_MSG_CHECKING([for pygtk h2def])
-	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.py"
+	PYGTK_H2DEF="$PYTHON `$PKG_CONFIG --variable=codegendir pygtk-2.0`/h2def.pyc"
 	AC_MSG_RESULT([$PYGTK_H2DEF])
 
 	AC_SUBST([PYGTK_DEFSDIR])
