--- eog-0.6/configure.in~	Thu Feb 15 03:03:34 2001
+++ eog-0.6/configure.in	Sun Aug 25 18:47:50 2002
@@ -8,7 +8,8 @@
 
 AM_MAINTAINER_MODE
 
-AM_PROG_XML_I18N_TOOLS
+AC_PROG_XML_I18N_TOOLS
+AC_PROG_INTLTOOL
 
 AM_ACLOCAL_INCLUDE(macros)
 
@@ -25,10 +25,6 @@
 ALL_LINGUAS="az ca cs da de el es et fi fr ga gl hu it ja ko lt nl no pl pt_BR ro ru sl sv tr uk zh_CN.GB2312 zh_TW.Big5"
 AM_GNU_GETTEXT
 
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-
 AM_PATH_GDK_PIXBUF(0.9.0, ,AC_MSG_ERROR([Need gdk-pixbuf-0.9.0 or later!]))
 
 AM_PATH_GCONF(0.12,,,gconf-gtk)
@@ -141,9 +137,8 @@
 dnl Test for libjpeg
   if test -z "$LIBJPEG"; then
     AC_CHECK_LIB(jpeg, jpeg_destroy_decompress,
-      jpeg_ok=yes,
-      jpeg_ok=no
-      AC_MSG_WARN(*** JPEG saving code will not be built (JPEG library not found) ***))
+      [jpeg_ok=yes],
+      AC_MSG_WARN([*** JPEG saving code will not be built (JPEG library not found) ***]))
     if test "$jpeg_ok" = yes; then
       AC_MSG_CHECKING([for jpeglib.h])
       AC_TRY_CPP(
@@ -151,16 +146,15 @@
 #undef PACKAGE
 #undef VERSION
 #include <jpeglib.h>],
-        jpeg_ok=yes,
-        jpeg_ok=no)
+        [jpeg_ok=yes])
       AC_MSG_RESULT($jpeg_ok)
       if test "$jpeg_ok" = yes; then
         LIBJPEG='-ljpeg'
         AC_CHECK_LIB(jpeg, jpeg_simple_progression,     
-          AC_DEFINE(HAVE_PROGRESSIVE_JPEG),
-          AC_MSG_WARN(JPEG library does not support progressive saving.))
+          [AC_DEFINE(HAVE_PROGRESSIVE_JPEG)],
+          [AC_MSG_WARN([JPEG library does not support progressive saving.])])
       else
-          AC_MSG_WARN(*** JPEG saving code will not be built (JPEG header file not found) ***)
+          AC_MSG_WARN([*** JPEG saving code will not be built (JPEG header file not found) ***])
       fi
     fi
   fi
@@ -172,24 +166,23 @@
 dnl Test for libpng
   if test -z "$LIBPNG"; then
     AC_CHECK_LIB(png, png_read_info,
-      AC_CHECK_HEADER(png.h,
-        png_ok=yes,
-        png_ok=no),
-      AC_MSG_WARN(*** PNG saving code will not be built (PNG library not found) ***), -lz -lm)
+      [AC_CHECK_HEADER(png.h,
+        [png_ok=yes],
+        [png_ok=no])],
+      AC_MSG_WARN([*** PNG saving code will not be built (PNG library not found) ***]))
     if test "$png_ok" = yes; then
       AC_MSG_CHECKING([for png_structp in png.h])
       AC_TRY_COMPILE([#include <png.h>],
         [png_structp pp; png_infop info; png_colorp cmap; png_create_read_struct;],
-        png_ok=yes,
-        png_ok=no)
+        [png_ok=yes])
       AC_MSG_RESULT($png_ok)
       if test "$png_ok" = yes; then
         PNG='png'; LIBPNG='-lpng -lz'
       else
-        AC_MSG_WARN(*** PNG saving code will not be built (PNG library is too old) ***)
+        AC_MSG_WARN([*** PNG saving code will not be built (PNG library is too old) ***])
       fi
     else
-     AC_MSG_WARN(*** PNG saving code will not be built (PNG header file not found) ***)
+     AC_MSG_WARN([*** PNG saving code will not be built (PNG header file not found) ***])
     fi
   fi
 if test "x$png_ok" = "xyes" ; then
@@ -199,44 +192,11 @@
 AC_SUBST(LIBJPEG)
 AC_SUBST(LIBPNG)
 
-
 dnl ******************************
 dnl More compiler warnings
 dnl ******************************
 
-AC_ARG_ENABLE(more-warnings,
-[  --enable-more-warnings  Maximum compiler warnings],
-set_more_warnings="$enableval",[set_more_warnings=no])
-warning_flags=
-realsave_CFLAGS="$CFLAGS"
-AC_MSG_CHECKING(for more warnings, including -Werror)
-if test "$GCC" = "yes" -a "$set_more_warnings" != "no"; then
-	AC_MSG_RESULT(yes)
-	warning_flags="-Wall -Wchar-subscripts -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wpointer-arith -Werror"
-	CFLAGS="$warning_flags $CFLAGS"
-
-	for option in -Wsign-promo -Wno-sign-compare; do
-		SAVE_CFLAGS="$CFLAGS"
-		CFLAGS="$CFLAGS $option"
-		AC_MSG_CHECKING([whether gcc understands $option])
-		AC_TRY_COMPILE([], [],
-			has_option=yes,
-			has_option=no,)
-		CFLAGS="$SAVE_CFLAGS"
-		AC_MSG_RESULT($has_option)
-		if test $has_option = yes; then
-		  warning_flags="$warning_flags $option"
-		fi
-		unset has_option
-		unset SAVE_CFLAGS
-	done
-	unset option
-else
-	AC_MSG_RESULT(no)
-fi
-CFLAGS="$realsave_CFLAGS"
-EXTRA_WARNING_CFLAGS="$warning_flags"
-AC_SUBST(EXTRA_WARNING_CFLAGS)
+GNOME_COMPILE_WARNINGS
 
 AC_OUTPUT([
 Makefile
