diff -urN eog-2.26.2/src/Makefile.am eog-2.26.2.new/src/Makefile.am
--- eog-2.26.2/src/Makefile.am	2009-05-18 21:37:55.000000000 +0200
+++ eog-2.26.2.new/src/Makefile.am	2009-05-20 18:20:57.411739872 +0200
@@ -117,8 +117,17 @@
 
 if HAVE_EXIF
 INST_H_FILES +=				\
-	eog-exif-util.h			\
 	eog-exif-details.h
+else
+if HAVE_EXEMPI
+INST_H_FILES +=				\
+	eog-exif-details.h
+endif
+endif
+
+if HAVE_EXIF
+INST_H_FILES +=				\
+	eog-exif-util.h
 libeog_la_SOURCES +=			\
 	eog-exif-util.c			\
 	eog-exif-details.c
@@ -133,8 +142,6 @@
 endif
 
 if HAVE_EXEMPI
-INST_H_FILES +=				\
-	eog-exif-details.h
 libeog_la_SOURCES +=			\
 	eog-exif-details.c
 endif
