--- eog-3.28.0/doc/reference/meson.build.gtkdoc	2018-03-14 21:25:17.440841661 +0100
+++ eog-3.28.0/doc/reference/meson.build	2018-03-14 21:25:30.537508328 +0100
@@ -32,7 +32,7 @@ configure_file(
   configuration: version_conf
 )
 
-doc_path = join_paths(eog_datadir, 'gtk-doc', 'html', meson.project_name())
+doc_path = join_paths(eog_datadir, 'doc', 'gtk-doc', 'html', meson.project_name())
 
 gnome.gtkdoc(
   meson.project_name(),
