diff -aurN eog-2.13.90.orig/data/eog.desktop.in.in eog-2.13.90/data/eog.desktop.in.in
--- eog-2.13.90.orig/data/eog.desktop.in.in	2006-01-26 06:17:59.000000000 +0100
+++ eog-2.13.90/data/eog.desktop.in.in	2006-01-31 00:04:10.020754000 +0100
@@ -8,7 +8,7 @@
 StartupNotify=true
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Graphics;RasterGraphics;Viewer;
+Categories=GTK;GNOME;Graphics;Viewer;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=EOG
 X-GNOME-Bugzilla-Component=general
