--- eog-41.0/meson.build.orig	2021-09-18 23:35:28.033842600 +0200
+++ eog-41.0/meson.build	2021-09-25 11:35:55.586686479 +0200
@@ -206,7 +206,7 @@ config_h.set('HAVE_INTROSPECTION', enabl
 # RSVG (optional for scaling svg image)
 enable_librsvg = get_option('librsvg')
 if enable_librsvg
-  common_deps += dependency('librsvg-2.0', version: '>= 2.44.0')
+  common_deps += dependency('librsvg-2.0', version: '>= 2.40.0')
 endif
 config_h.set('HAVE_RSVG', enable_librsvg)
 
